src/style.css

```diff
@@ -1 +1,5 @@
 /* styles go here */
+
+span {
+  display: inline;
+}
```

Comment:
do you really need this?

Reply:
I don't know, I don't want to break something) I tried to connect styles and failed the tests, so it expects just text, and if I leave an empty document, I have an error "Unexpected empty source". Maybe I need to write something special?

Reply:
you can remove this file at all since you don't need to style anything

Reply:
don't be afraid to experiment

Reply:
Understood, thank you)